Makefile

```diff
@@ -40,6 +40,19 @@ endif
 build:
 	DOCKER_BUILDKIT=1 docker build --tag $(IMAGE) --target $(DOCKER_IMAGE_TAG)  --build-arg record=$(record) .
 
+build_venv:
+	python3 -m venv mava_env
+	mava_env/bin/pip install -r jax_distribution/requirements.txt
+	mava_env/bin/pip install jax==0.4.9
+	mava_env/bin/pip install jaxlib==0.4.7+cuda11.cudnn82  -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
```

Comment:
I am not sure how we could make this a bit more general. Maybe we should support multiple builds?

Reply:
Same here maybe for now we can add a comment to check jax, jaxlib ver... but we can add it as a todo for later enhancements

Reply:
yeah maybe todo is the right thing to do now